src/containers/Storage/StorageGroups/StorageGroups.scss

```diff
@@ -1,4 +1,8 @@
 .global-storage-groups {
+    &__vdisks-column {
+        overflow: visible; // to enable stacked disks overflow the row
```

Comment:
for myself: questionable, revise later

Reply:
Increased stack layers offset from 3ps to 4ps, now 3-items stack will also overflow the row. Need to keep this, although 2 donors are rare.